mcs/class/System/Mono.Btls/MonoBtlsContext.cs

```diff
@@ -220,7 +220,7 @@ public override void FinishHandshake ()
 
 		void InitializeConnection ()
 		{
```

Comment:
This section is unrelated, it's most likely a left-over from some earlier debugging.  Should take this out.